connect/eaas/runner/handlers/transformations.py

```diff
@@ -3,8 +3,12 @@
 #
 # Copyright (c) 2022 Ingram Micro. All Rights Reserved.
 #
+import inspect
 import logging
 
+from connect.eaas.core.inject.models import (
```

Comment:
Context for such application type should be the one from connect.eaas.core.models